app/views/workshops/show.html.haml

```diff
@@ -4,7 +4,12 @@
       %h2
         Workshop at #{@workshop.host.name}
         %br
-        %small #{humanize_date(@workshop.date_and_time, with_time: true)} #{@workshop.distance_of_time}
+        %small #{humanize_date(@workshop.date_and_time)} #{@workshop.distance_of_time}
+        #workshop-time.time
+          %i.material-icons
+            access_time
+          = @workshop.time
+          = @workshop.ends_at.present? ? l(@workshop.ends_at, format: :time) : ""
```

Comment:
Missing hyphen between start and end time - this is what I'm seeing 

---
![Workshop show](https://user-images.githubusercontent.com/1710795/67131521-ed7dad80-f1fc-11e9-81ff-1f5cdf2679a9.png)

Reply:
thank you I missed that, now is fixed